package/CHANGELOG

```diff
@@ -13,6 +13,8 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
+
+
 ??/??/?? IAlibay, BFedder, inomag
```

Comment:
The number of empty lines here shouldn't change, although your handle should go on the list at line 16/18.

I'll fix this whilst I update for merge conflicts.